types/musicmatch/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Remove all the rules from here and fix errors if any, new packages cannot use custom tslint but have to just extend dtslint json file